scripts/test_sandbox.sh

```diff
@@ -70,11 +71,11 @@ function finish {
 }
 trap finish EXIT
 
-# Call app-specific setup_governance and test scripts
-check_existence=$(ls $app_dir/test/test.sh 2>/dev/null || true)
-if [ -z "$check_existence" ]; then
-    failed "You are missing a test.sh script in your application."
-    exit 0
+testScript="$app_dir/test/test.sh"
+if [ ! -f "$testScript" ]; then
+    echo "💥📂 Test file $testScript not found."
+    exit 1
+else
+  "$app_dir/test/test.sh" --nodeAddress "${nodeAddress}" \
+    --certificate_dir "$certificate_dir"
```

Comment:
Can we include this change on `test_docker.sh` also?

Reply:
Done